app/serializers/web_hook_serializer.rb

```diff
@@ -0,0 +1,24 @@
+class WebHookSerializer < ApplicationSerializer
+  def attributes(*args)
+    data = super
+
+    object.specific.each do |o|
+      data[o.rubygem.name] ||= []
+      data[o.rubygem.name] << { "failure_count" => o.failure_count, "url" => o.url }
+    end
+
+    globals = object.global.to_a
+    return data unless globals.present?
+
+    data['all gems'] = []
+    globals.each do |g|
+      data['all gems'] << { "failure_count" => g.failure_count, "url" => g.url }
+    end
+
+    data
+  end
```

Comment:
Our webhook response is a bit weird:
```Ruby
  {"rails":
    [{"failure_count":0,"url":"http://example.com"},
     {"failure_count":0,"url":"http://example2.com"}],
  "fakeweb":
    [{"failure_count":0,"url":"http://example.com"}],
  "all gems":
    [{"failure_count":0,"url":"http://example.com"}]}
```

Key 'rails' or 'fake-web' is **value** of the `web_hook.rubygem` and the aggregates all the web_hooks(of user) registered with that gem. "all gem" is the key for all the global webhooks.
Calling `each_serializer` on `current_user.web_hooks` would not work because web_hooks of same gem need to be aggregated. Here I am aggregating in the overridden `attributes` function. It also handles the weird case where `key` name is dynamic (AMS doesn't support this(?)).

Alternatively, webhooks can be aggregated before passing it to serializer. Which is how it worked previously:
```Ruby
=> current_user.all_hooks
# {"rails"=>[#<WebHook id: 1, user_id: 1, url: "http://example.com", failure_count: 0, created_at: "2016-09-01 19:47:46", updated_at: "2016-09-01 19:47:46", rubygem_id: 15494>,
             #<WebHook id: 3, user_id: 1, url: "http://example2.com", failure_count: 0, created_at: "2016-09-02 07:25:02", updated_at: "2016-09-02 07:25:02", rubygem_id: 15494>],
#  "fakeweb"=>[#<WebHook id: 2, user_id: 1, url: "http://example.com", failure_count: 0, created_at: "2016-09-01 20:23:58", updated_at: "2016-09-01 20:23:58", rubygem_id: 15069>]}
```
I am not sure how can I serialize this to match current web_hook response :point_up: 

cc: @bf4